runtime/bcutil/ComparingCursor.cpp

```diff
@@ -212,10 +212,16 @@ ComparingCursor::writeSRP(UDATA srpKey, DataType dataType)
 					markUnEqual();
 				} else {
 					U_16 cpIndex = _srpKeyProducer->mapKeyToCfrConstantPoolIndex(srpKey);
-
-					if ( J9UTF8_LENGTH(utf8) != _classFileOracle->getUTF8Length(cpIndex) ) {
+					bool isInjected = false;
```

Comment:
instead of doings you could just check if (`cpIndex` > _classFileMaxCPIndex_), if so then skip the check

Reply:
Just to confirm, is classFileMaxCPIndex == ConstantPoolCount?